### DISCUSSION TITLE
Don't wait for video to load to launch GUI

### DISCUSSION THREAD

**Discussion by avidseeker** [mpv-player/mpv#14361] [Q&A] [answered]
This is a config line suggested from Arch Wiki to open mpv on newsboat RSS video entries:
```
macro y set browser "mpv %u" ; open-in-browser ; set browser "elinks %u"
```

When using this macro within newsboat, mpv takes a few seconds to load the video and then it launches the GUI. Is it possible to change this command so that it launches the GUI first, and then load the video (through yt-dlp, etc.)?

**Comment by kasper93** [comment, marked as answer]
[`--force-window=<yes|no|immediate>`](https://mpv.io/manual/master/#options-force-window)
